openfisca_web_api/app.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+import os
+from flask import Flask, jsonify, abort
+
+from model import build_parameters, build_tax_benefit_system, build_headers
+
+def create_app(country_package = os.environ.get('COUNTRY_PACKAGE')):
+    if country_package is None:
+        raise ValueError(u"You must specify a country package to start the api. For instance, COUNTRY_PACKAGE=openfisca_france flask run")
```

Comment:
Exception string must be encoded (in UTF-8), so either remove the `u` or add `.encode('utf-8')`